docs/guides/architecture/configuration-files.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 1
+---
+
+# Configuration Files
+
+Since Tauri is a toolkit for building applications there can be many files to configure project settings. Some common files that you may run across are `tauri.conf.json`, `package.json` and `Cargo.toml`. We briefly explain each on this page to help point you in the right direction for which file to modify.
+
+## `tauri.conf.json`
+
+This is the file used by the Tauri process. You can define build settings (such as the [command run prior to `tauri build`][before-build-command] or [`tauri dev`][before-dev-command]), set the [name and version of your app][package-config], [control the Tauri process][tauri-config], and configure any plugin settings. You can find all of the options in the [`tauri.conf.json` API reference].
```

Comment:
```suggestion
This is the file used by the Tauri process. You can define build settings (such as the [command run prior to `tauri build`][before-build-command] or [`tauri dev`][before-dev-command]), set the [name and version of your app][package-config], [control the Tauri process][tauri-config], and configure any plugin settings. You can find all of the options in the [`tauri.conf.json` API reference].

:::note

The Tauri config can be a `.json`, `.json5`, or `.toml` format. The structure and values are the same across all formats, however the formatting should be consistent with the respective file format.

Note that `Tauri.toml` is only supported for Tauri 1.1.0 and above.

:::
```

Reply:
This should probably mention that json5 and toml have their own feature flags _which are disabled by default_. The flags are `config-json5` and `config-toml`

Reply:
Where do those flags go? In the `Cargo.toml` file?

Reply:
Yeah on the tauri dependency, like this thingy here https://github.com/tauri-apps/tauri/blob/dev/examples/sidecar/src-tauri/Cargo.toml#L14

Reply:
And added to the regular dependency, not the build dependency, right?

Reply:
yeah

Reply:
Actia;;y tauri-build has that flag too, but i've never seen it being used 🤔 Gonna double check if it's enough on tauri